src/query_parser/parser.y

```diff
@@ -2,7 +2,7 @@
 %left TILDE.
 %left TAGLIST.
 %left QUOTE.
-%left COLON.
+%left MODIFIER.
```

Comment:
`MODIFIER` is a terminal token and it is not used in any rule beside the one `modifier ::= MODIFIER`.
Previously it was used in the `modifier COLON expr` rule, probably because some conflicts, but now we don't need it and we can use `COLON` for this rule (like we use it in the rest of the modifier rules).